app/presenters/organisations_presenter.rb

```diff
@@ -1,4 +1,6 @@
 class OrganisationsPresenter
```

Comment:
You might want to move this file to `app/presenters/organisations/index_presenter.rb` to fit with the changes I'm making. Having all the presenters in an organisations directory means we don't have to keep putting 'organisations' at the start of all the filenames.

Reply:
Good point. Done 👍